broadcast/errors.go

```diff
@@ -21,9 +21,6 @@ var ErrClientUndefined = errors.New("client is undefined")
 // ErrAllBroadcastersFailed is returned when all configured broadcasters failed to query or broadcast the transaction.
 var ErrAllBroadcastersFailed = errors.New("all broadcasters failed")
 
-// ErrURLEmpty is returned when the API URL is empty.
-var ErrURLEmpty = errors.New("url is empty")
-
```

Comment:
Why did you remove it?

Reply:
I moved it to the http_client to release the client from `broadcast` package dependency.